kots-exporter/kots-exporter.sh

```diff
@@ -357,6 +357,7 @@ postgres_migration(){
 
     kubectl delete statefulsets.apps postgresql --namespace $namespace --cascade=orphan
     kubectl delete secret postgresql --namespace $namespace
+    fi
```

Comment:
This snuck into your PR by accident I imagine

Reply:
thanks, removed it